### PR TITLE
Podman controller

### PR DESCRIPTION
**Pull request by fgiorgetti**

### CODE REVIEW

**Review by grs — APPROVED** [review]

File: pkg/images/images.go [hunk 1]
```diff
@@ -4,6 +4,7 @@ const (
 	DefaultImageRegistry       string = "quay.io/skupper"
 	RouterImageName            string = "skupper-router:main"
 	ServiceControllerImageName string = "service-controller:master"
+	ControllerPodmanImageName  string = "controller-podman:master"
```
Comment (nluaces) [inline pkg/images/images.go:7]: should we add it into the version command output?

Comment (nluaces) [reply pkg/images/images.go:7]: I take that back, it is probably better to add it when the update command implementation for podman is ready.

**Comment by nluaces** [comment]
Looks good to me! I tested it on local and I could see my podman site on the console and its services.

**Review by nluaces — APPROVED** [review]

**Review by ajssmith — APPROVED** [review]
lgtm - parent site succesfully linked

**Pull request merged**